### DISCUSSION TITLE
万科海上明月这个是配套没兑现啊，能不能核实认真一点

### DISCUSSION THREAD

**Discussion by gofunbox** [WeNeedHome/SummaryOfLoanSuspension#542] [General]
万科海上明月这个是配套没兑现啊，能不能核实认真一点

**Comment by zxcs001** [comment]
认真一点？人家很认真的好不好，政府的任务就是糊弄过去，能少花一分钱就少花一分钱，让这帮房地产多挺一天是一天，晚爆一天是一天。

**Reply by gofunbox** [reply]
全国各省市烂尾楼停贷通知汇总 不是么 万科海上明月是楼已经交付 配套没有兑现 严谨一点有错么 还是要刷数据啊

**Reply by SuperBert** [reply]
按你的意思 楼盘交付了 噪音超标也是停贷的理由？？活着多累啊，早死早超生吧

**Reply by TALK1949** [reply]
> 全国各省市烂尾楼停贷通知汇总 不是么 万科海上明月是楼已经交付 配套没有兑现 严谨一点有错么 还是要刷数据啊

为啥一处错误就能让你联想到刷数据？
为什么不用这种严谨的态度要求政府公权力呢？
你又不给人家汇总数据的义工交税，有错误提出就可以了，上纲上线反而让你看起来动机就有问题。

**Comment by SuperBert** [comment]
这个就不该收录进来，纯属业主作死
